modules/migration-debugging-velero-admin-webhooks.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+[id="migration-debugging-velero-admin-webhooks_{context}"]
+= Issues with Velero and admission webhooks
+
+Velero has limited abilities to resolve admission webhook issues during a restore, so you might need to use an additional Velero plug-in or make changes to how you restore the workload.
+
+Typically workloads with admission webhooks require you to create a resource of a specific kind first.
+
+Documented restore workarounds:
+
+* link:https://github.com/knative/serving/issues/10926#issuecomment-886434177[Knative] requires you to resrore the top level `Service` resource be restored first by entering the following command:
++
+[source,terminal]
+----
+$ velero restore <restoreName> --from-backup=<backupName> --include-resources \
+  service.serving.knavtive.dev
+----
+
+* IBM AppConnect resources requires special care in link:https://github.com/openshift/oadp-operator/issues/538[configuration] to be restored properly with Velero.
+
+For more information on admission plug-ins and webhooks, see xref:../../architecture/admission-plug-ins.adoc[admission plug-ins], especially xref:../../architecture/admission-plug-ins#admission-webhooks-about_admission-plug-ins[webhook admission plug-ins] and xref:../../architecture/admission-plug-ins#admission-webhook-types_admission-plug-ins[types of webhook admission plug-ins].
```

Comment:
@RichardHoch 
We cannot have xrefs in modules. Make it into 'Additional resources' and a 'trailing include' in the Troubleshooting assembly following the include statement for this module. And in additional resources, we are now supposed to limit the text to the words in the square brackets. So this paragraph should appear in the Troubleshooting assembly like this:

include::modules/migration-debugging-velero-admin-webhooks.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources 
* xref:../../architecture/admission-plug-ins.adoc[Admission plug-ins]
* xref:../../architecture/admission-plug-ins#admission-webhooks-about_admission-plug-ins[Webhook admission plug-ins]
* xref:../../architecture/admission-plug-ins#admission-webhook-types_admission-plug-ins[Types of webhook admission plug-ins]